daemon/media_player.c

```diff
@@ -1104,6 +1104,57 @@ static int media_player_find_file_begin(struct media_player *mp) {
 static bool media_player_read_packet(struct media_player *mp) {
 	if (!mp->coder.fmtctx)
 		return true;
+	// Handle raw RTP file playback
+    if (mp->coder.audio_raw_rtp_mode) {
```

Comment:
Since `media_player_read_packet` is already used as `run_func` function pointer, this whole block would lend itself to also be its own function and then be used as `run_func`, to avoid the if/else